modules/rosa-getting-started-deleting-a-cluster.adoc

```diff
@@ -5,7 +5,12 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="rosa-getting-started-deleting-a-cluster_{context}"]
+ifdef::openshift-rosa-hcp[]
```

Comment:
These two conditions seem to be superfluous. Whether the distro is openshift-rosa-hcp or not, the title is the same.